x-pack/plugins/cases/server/telemetry/index.ts

```diff
@@ -42,7 +43,7 @@ export const createCasesTelemetry = async ({
 }: CreateCasesTelemetryArgs) => {
   const getInternalSavedObjectClient = async (): Promise<ISavedObjectsRepository> => {
     const [coreStart] = await core.getStartServices();
-    return coreStart.savedObjects.createInternalRepository(SAVED_OBJECT_TYPES);
+    return coreStart.savedObjects.createInternalRepository([...SAVED_OBJECT_TYPES, FILE_SO_TYPE]);
```

Comment:
Just calling this out, to retrieve the average file size we have to do an average aggregation over the file service's saved objects that contain a metadata field `caseId`. Hopefully it's ok that we're accessing the file service's saved object internals.

The advantage of doing it this way is that we don't need to duplicate the file size within case comments.